webservices/tasks/refresh.py

```diff
@@ -32,13 +32,31 @@ def refresh_and_rebuild():
     As with regular updates, email logs are also sent.
     """
     buffer = io.StringIO()
+    itemized_schedules = ['a', 'b']
+
     with mail.CaptureLogs(manage.logger, buffer):
         try:
-            manage.rebuild_aggregates()
-            manage.refresh_materialized()
+            projected_weekly_totals = manage.get_projected_weekly_itemized_totals(
+                itemized_schedules
+            )
+            manage.update_all()
```

Comment:
Okay, we're good here.  `update_all()` runs all of the scripts in `data/sql_setup` when it hits the `update_itemized()` method calls.  The schedule A and B scripts still take care of dropping and recreating the queue tables in this case, so yes, they are taken care in this case already.